ledfx/effects/concentric.py

```
@@ -0,0 +1,131 @@
+
+    NAME = "Concentric"
+    CATEGORY = "2D"
+
+    HIDDEN_KEYS = [
+        "background_brightness",
+        "mirror",
+        "flip",
+        "dump",
+        "gradient_roll",
+        "rotate",
+    ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)